### ISSUE TITLE
Extreme Reactors fluids not working as a "Fluid" in bucket form

### ISSUE THREAD

**Issue by jzburda** [ZeroNoRyouki/ExtremeReactors2#160]
Problem: Any extreme reactors fluidized fluid (Veridium and Magentite tested specifically) is not working as a fluid in bucket form. 
This has problems such as:
Not being able to be *bucketed* into ANY kind of tank specifically. This includes back into a RS fluid grid. This has been tried on an Evilcraft Dark Tank, A Mekanism Portable Tank, and a EnderIO fluid tank, all do not work. 
Unable to be used as a fluid encode in Refined Storage
etc.

Extreme Reactors  1.20.1-2.0.83
Forge 47.2.20
Modpack: All The Mods 9 -  0.3.0

**Comment by ZeroNoRyouki** [comment]
Thank you for the report.

I found a problem that was introduced when I fixed the fluid colour on the bucket items and now the buckets can be used again to drop and pickup the fluids from ER, including inserting them in a fluid tank from Mekanism. 

Does RS uses "normal" buckets to do what you described? Can't test it right now

**Comment by jzburda** [comment]
> Thank you for the report.
> 
> I found a problem that was introduced when I fixed the fluid color on the bucket items and now the buckets can be used again to drop and pickup the fluids from ER, including inserting them in a fluid tank from Mekanism.
> 
> Does RS uses "normal" buckets to do what you described? Can't test it right now

Yes, it does. It works with vanilla buckets with fluid in them, and most modded fluids in bucket form.
I forgot to mention this, but this youtube video will visually document both parts of the issue. (And the intended function of what's supposed to happen with encoding a fluid into a Refined Storage fluid-compatible GUI) https://youtu.be/u7leAKKbwJg (Partially with the Mek fluid tank, I forgot that I had 1 bucket of verderium in the tank before visually explaining the issue, because as it turns out, using the Mekanism Fluid tank in bucket mode allows for the fluid to go in when placed on the ground, but I didn't say that part because that's not part of the actual issue at hand, but could be useful information)

Edit: It does go in through a fluid pipe so I might be dumb on this one)
AND to add on to the issue, all your fluids seem to be working correctly for putting into a sophisticated backpack tank (Which is what I'm currently using as a two-step system to automate the reprocessor) EXCEPT for rossinite. This may have already been fixed with what you've implemented in your previous post. If you further need to test this, the schematic is as follows
-------------------------------------------------------------------------------------Fluid Pipe ( I used Mekanism) -> Fluid input
Crafter -/> Sophisticated Backpack with Tank Upgrade with Cobblestone in all but one slot  -V^
-----------------------------------------------------------------------------Item pipe (Logistical Sorter - > Logistical Pipe) V 
--------------------------------------------------------------------------------------------------------------------------Item Input
